src/PRAS2PowerSystems.jl

```diff
@@ -1,240 +1,265 @@
 """
-    generate_outage_profile(pras_system,num_runs,psy_sys,num_scenarios,location)
+    generate_outage_profile!(
+        sys::PSY.System,
+        aggregation::Type{AT},
+        method::PRASCore.SequentialMonteCarlo,
+    ) where {AT <: PSY.AggregationTopology, RM <: PRASCore.Results.ReliabilityMetric}
 
-Process the assess results to get timeseries of generator status and include
-this timeseries data to the corresponding component in PSY System and exported
-using to_json method (serializing the PSY System).
-
-...
+Analyze resource adequacy using Monte Carlo simulation and add the asset status from the worst sample
+to PSY.TimeSeriesForcedOutage of the component.
 
 # Arguments
 
-  - `pras_system::PRASCore.SystemModel`: PRAS System
-  - `num_runs::Int64`: Number of PRAS runs
-  - `psy_sys::PSY.System`: PSY System
-  - `num_scenarios::Int64`: Number of scenarios of user interest.
-  - `location::String`: Location to store outage profile.
-    ...
-
-# Examples
-
-```julia-repl
-julia> generate_outage_profile(results, pras_sys, psy_sys, 1)
-PSY System exported using to_json method in InfrastructureSystems
-```
-"""
-function generate_outage_profile(
-    pras_system::PRASCore.SystemModel,
-    psy_sys::PSY.System;
-    location::Union{Nothing, String}=nothing,
-    num_runs::Union{Nothing, Int}=nothing,
-    num_scenarios::Union{Nothing, Int}=nothing,
-)
+  - `sys::PSY.System`: PowerSystems.jl system model
+  - `aggregation::Type{AT}`: Aggregation topology to use in translating to PRAS
+  - `method::PRASCore.SequentialMonteCarlo`: Simulation method to use
 
-    #kwargs handling
-    if (location === nothing)
-        location = dirname(dirname(@__DIR__))
-        @warn "Location to save the exported PSY System not specified. Using the data folder of the module."
-    end
+# Returns
 
-    if (num_runs === nothing)
-        num_runs = 10000
-        @warn "Number of samples of PRAS Assess not specified. Using 10,000 samples."
-    end
+  - PSY System with PSY.TimeSeriesForcedOutage for all components for which asset status is available
+"""
+function generate_outage_profile!(
+    sys::PSY.System,
```

Comment:
This one should call the other `generate_outage_profile!` with the template and all that.

Reply:
Done.